src/makielayout/blocks/toggle.jl

```diff
@@ -44,6 +44,14 @@ function initialize_block!(t::Toggle)
     button = scatter!(topscene, buttonpos, markersize = buttonsize,
         color = t.buttoncolor, strokewidth = 0, inspectable = false, marker = Circle)
 
+    onany(topscene, t.orientation, t.layoutobservables.computedbbox) do angle_rad, bbox
```

Comment:
`orientation` should also support `:horizontal` or `:vertical` like the rest of them IMO.  Maybe just have a function that converts from the symbol to the angle, and invoke that within the statement?